lib/iris/tests/integration/plot/test_vector_plots.py

```diff
@@ -188,19 +180,44 @@ def _nonlatlon_xyuv():
         return x, y, u, v
 
     @staticmethod
-    def _latlon_uv_cubes(grid_cube):
+    def _latlon_uv_cubes(grid_cube, scale_factor=None):
         # Increase the range of wind speeds used in the barbs test to test all
         # barbs shapes
         u_cube, v_cube = MixinVectorPlotCases._latlon_uv_cubes(grid_cube)
-        scale_factor = 30
+        if scale_factor is None:
+            scale_factor = 30
         u_cube.data *= scale_factor
         v_cube.data *= scale_factor
         return u_cube, v_cube
 
     def plot_function_to_test(self):
         return barbs
 
+    def test_2d_plain_latlon_on_polar_map(self):
+        # Test 2d vector plotting onto a different projection.
+
+        # use a scale factor that ensures the barb vector magnitudes fall
+        # sufficiently within the default matplotlib barb bins in order to
+        # avoid floating point issues where barbs flip/flop between bins.
```

Comment:
I think it would help Future Us to have a comment on why this is only done for polar.  If the reference images have to be regenerated for some other reason in future, it would make sense to have them all use 30.1 (as you did initially) and rationalise the code back down.

Reply:
Yup. TBH I was just being lazy and losing the will to live ...

I'll update the comment ... I'd be interested to see how volatile these tests are, so yeah, I'll add a note to our future selves and respin the barbs images with a new common `scale_factor` and backout the specialization code :+1: